v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java

```diff
@@ -312,6 +314,7 @@ public interface TokenizePipelineOptions extends DataflowPipelineOptions {
 
     @TemplateParameter.Text(
         order = 3,
+        groupName = "Source",
```

Comment:
I think this is not related to the source and it's a transformation applied in the middle, I'd leave it ungrouped.